src/main/java/org/opentripplanner/model/UpdateError.java

```diff
@@ -48,6 +48,7 @@ public enum UpdateErrorType {
     INVALID_STOP_SEQUENCE,
     NOT_IMPLEMENTED_UNSCHEDULED,
     NOT_IMPLEMENTED_DUPLICATED,
+    NOT_MONITORED,
```

Comment:
So NOT_MONITORED is an error, not a warning?

Reply:
It is a borderline case. I've set all cases, where we don't apply any updates to be errors, and where we apply at least part of the update to be a warning.